src/auth/controllers/auth.controller.spec.ts

```diff
@@ -58,4 +66,44 @@ describe('AuthController', () => {
       expect(await authController.login(reqObject, loginInputDto)).toBe(null);
     });
   });
+
+  describe('refreshToken', () => {
+    let mockTokenUser: TokenUserIdentity;
+    let mockRefreshTokenInputDto: RefreshTokenInput;
+    let mockAuthToken: AuthTokenOutput;
+    let mockRequest: any;
+
+    beforeEach(() => {
+      mockTokenUser = { id: 1 };
+      mockRefreshTokenInputDto = plainToClass(RefreshTokenInput, {
+        refreshToken: 'mock_refrsh_token',
+      });
+      mockAuthToken = {
+        accessToken: 'new_access_token',
+        refreshToken: 'new_refresh_token',
+      };
+      mockRequest = {
+        user: mockTokenUser,
+      };
+
+      jest
+        .spyOn(mockedAuthService, 'refreshToken')
+        .mockImplementation(async () => mockAuthToken);
+    });
+
+    it('should generate refresh token', async () => {
+      await validateOrReject(mockRefreshTokenInputDto);
```

Comment:
```
      await validateOrReject(mockRefreshTokenInputDto);
```
I think this was added by me in the controller layer in the beginning, but after giving another thought, maybe this `validateOrReject` is not needed after all.
I will follow this up with another PR and remove it from all controller tests. :bow:

Reply:
kept it as you said - you remove all once decided